hugo/content/about.md

```diff
@@ -0,0 +1,73 @@
+---
+title: About
+toc: true
+---
+
+# About the project
+
+Mumble is a free, open source, low latency, high quality voice chat application.
+
+## Source Code & Development
+
+You find the Source Code and further instructions regarding Building and Development in our [GitHub Repo](https://github.com/mumble-voip).
+
+## Licenses
+
+<!-- https://github.com/mumble-voip/mumble/blob/master/LICENSE -->
+
+<!-- more licenses? should be as correct and complete as possible -->
+
+# Who we are
+
+<!-- Title could also be: Developers, Team etc. -->
+
+<!-- improve -->
+
+We are a team of voluntary developers - both professional and non-professional coders - from different countries - including Germany, Italy, USA and more.
+
+<!--
+Link to Github Team page:
+https://github.com/orgs/mumble-voip/people
+
+-->
+
+<!-- Distinction between active and non-active developers? -->
+
+<!-- could contain more blabla about vision, free-software dedication etc. -->
+
+
+<!-- Credits/List of Contributors ? -->
+
+<!-- https://www.mumble.info/AUTHORS -->
+
+<!-- see also: "contribute" below -->
+
+# About the website
+
+<!-- link to github repo 
+
+https://github.com/mumble-voip/mumble-www
+-->
+
+<!-- information on who operates the website etc. -->
+
+
+
+<!-- add category? 
+# Contribute
```

Comment:
Having this section on the `About` page seems a bit odd anyways :thinking:

Reply:
You are right, we have the general contribute page anyways, so it's visible on first sight.
I was just looking at other websites and also found this.